### PR TITLE
Add sudo prompt & update README

### PR DESCRIPTION
**Pull request by david-martin**
This change adds the `--ask-become-pass` arg for setup & reset scripts.
This is to prevent below error

```
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "sudo: a password is required\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
```

Instructions are also updated to format a command, and add a
troubleshooting section for an issue with docker-py version

### CODE REVIEW

File: local/linux/README.md [hunk 1]
```diff
@@ -47,9 +48,24 @@ To terminate the local instance run the below
...
 To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
-  * `cd local`
+  * `cd local/linux`
   * `./reset_environment.sh`
 
 ### Tested with
   * ansible 2.2.2.0 & 2.3.0.0
     * Problems were seen using ansible 2.0
+
+### Troubleshooting
```
Comment (jwmatthews) [inline local/linux/README.md]: +1 to a Troubleshooting section, good idea.

File: local/linux/README.md [hunk 2]
```diff
@@ -47,9 +48,24 @@ To terminate the local instance run the below
...
+
+```
+Error pulling image docker.io/ansibleplaybookbundle/ansible-service-broker-apb:summit - pull() got an unexpected keyword argument 'decode'
+```
+
+This is a problem with the version of docker-py. More info in https://github.com/ansible/ansible-modules-core/issues/5515.
+Update to version 1.10.6 of docker-py
+
+```
+<sudo> pip install --upgrade docker-py
```
Comment (jwmatthews) [inline local/linux/README.md]: Let's remove the suggestion to upgrade docker-py, instead let's recommend to remove 'docker-py' in favor of 'docker' pip module.

We've seen other errors related to having docker-py being present:
 https://github.com/fusor/catasb/issues/52 

We used to enforce the upgrade to 1.10.6 of docker-py, then we ended up upgrading to 'docker'.  The install of 'docker' pip module is handled here:

https://github.com/fusor/catasb/blob/master/ansible/roles/openshift_setup/tasks/main.yml#L100

**Comment by david-martin** [comment]
@jwmatthews Thanks for the feedback. I've updated the troubleshooting section to suggest uninstalling `docker-py` if that error is seen

**Comment by jwmatthews** [comment]
LGTM 

@david-martin thank you for the PR

**Pull request merged**